packages/apps-config/src/api/spec/index.ts

```diff
@@ -53,6 +54,7 @@ export default {
   'encointer-node-teeproxy': encointerNodeTeeproxy,
   'encointer-parachain': encointerPara,
   hanonycash,
+  'hack-hydra-dx': hydrate,
```

Comment:
Can you just move this one up (alphabetical). 

Here it does make sense since it makes it easier to maintain, quite a bit going in. 

(Sorry, didn’t see this complaint initially on CI)

Reply:
sure, it was hydrate originally so it was sorted correctly 🤦